src/appleseed/foundation/meta/tests/test_beziercurve.cpp

```diff
@@ -411,7 +411,9 @@ TEST_SUITE(Foundation_Math_BezierCurveIntersector)
             1.0f
         };
         Matrix4f Basis = Matrix4f::from_array(BezierInverseBasisArray) * Matrix4f::from_array(BSplineBasisArray);
-        const BezierCurve3f Curves[] = { BezierCurve3f(BezierCurve3f(ControlPoints, Widths, Opacities, Colors), Basis, true) };
+        BezierCurve3f Curve = BezierCurve3f(BezierCurve3f(ControlPoints, Widths, Opacities, Colors));
```

Comment:
We can remove now one of the BezierCurve3f. Same thing bellow.

Reply:
Fixed.